packages/blocks/src/api/serializer.js

```diff
@@ -198,7 +198,7 @@ export function getSaveContent( blockTypeOrName, attributes, innerBlocks ) {
  */
 export function getCommentAttributes( blockType, attributes ) {
 	return reduce(
-		blockType.attributes,
+		blockType?.attributes,
```

Comment:
To match the style of this package, if we're passing blockType as an argument from `getBlockType( blockName )`, let's update that instead of adding sanitization in this function.

Reply:
@gwwar can you explain this change? I'm not clear with your comment.

Reply:
Functions in this file do not sanitize passed arguments (which looks like a code-style choice.) This means callers should be smart about calling `getCommentAttributes`. Eg they should only call this function when blockType is well defined. 

This pushes the check out of this function and anywhere we might call `getCommentAttributes`. Overall it's fine to drop this change and maybe look for those in a follow up PR.